docs/essentials/exceptions/exceptions-create-user-defined.md

```diff
@@ -1,6 +1,6 @@
 # How to create user-defined exceptions
```

Comment:
I was wondering if the topics under Exception Handling Fundamentals shouldn't be combined into one